src/redux/rockets/rocketSlice.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] You have done really well in implementing the changes requested by the previous code reviewer, :clap: , However, as she mentioned, `"Please kindly do not disable linters, fixing linters helps us to write a strong and clean code base"` 😁, Please resolve that.